state-chain/chains/src/btc/api.rs

```diff
@@ -149,7 +149,8 @@ impl<E: ReplayProtectionProvider<Bitcoin>> ExecutexSwapAndCall<Bitcoin> for Bitc
 // transfer_fallback is unsupported for Bitcoin.
 impl<E: ReplayProtectionProvider<Bitcoin>> TransferFallback<Bitcoin> for BitcoinApi<E> {
 	fn new_unsigned(_transfer_param: TransferAssetParams<Bitcoin>) -> Result<Self, DispatchError> {
-		Err(DispatchError::Other("Bitcoin's TransferFallback is not supported."))
+		// Bitcoin's TransferFallback is not supported.
+		Err(DispatchError::Other(""))
```

Comment:
Maybe I'm missing something, but what does this improve exactly?

Reply:
Changed this to a proper typed error enum